src/components/MessageInput/MessageInput.tsx

```diff
@@ -259,22 +256,16 @@ export const MessageInputWithContext = <
     attachmentPickerBottomSheetHeight,
     attachmentSelectionBarHeight,
     bottomInset,
-    closePicker,
-    openPicker,
     selectedImages,
     selectedPicker,
     setMaxNumberOfFiles,
     setSelectedImages,
-    setSelectedPicker,
   } = useAttachmentPickerContext();
 
   useEffect(() => {
     setMaxNumberOfFiles(maxNumberOfFiles ?? 10);
 
-    return () => {
-      closePicker();
-      setSelectedPicker(undefined);
-    };
+    return closeAttachmentPicker;
```

Comment:
hmmm maybe check with @nhannah if there was a reason the order was reversed in the cleanup of this effect vs just using the `closeAttachmentPicker` function which has the calls in the inverse order

Reply:
Checked with him :)